subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -125,6 +125,8 @@ dependencies {
         api(libs.testng)                { version { strictly("6.3.1"); because("later versions break test cross-version test filtering") }}
         api(libs.tomlj)                 { version { strictly(tomljVersion) }}
         api(libs.trove4j)               { version { strictly("1.0.20181211") }}
+        // TODO upgrade this AGP version to recent version
```

Comment:
Actually, we should drop our Android studio sync and use the real thing with the new features in Gradle profiler. Though that may be hard.